olm/build.gradle.kts

```diff
@@ -37,7 +37,19 @@ if (HostManager.hostIsMingw) {
     }
 }
 
+var kotlinCoroutines = "1.6.1"
```

Comment:
Can you get this version from [here](https://github.com/Dominaezzz/matrix-kt/blob/32cce5678b9a88a7033b4f4a37121d5fe6983bdf/build.gradle.kts#L13) instead?

[This](https://github.com/Dominaezzz/matrix-kt/blob/master/client/build.gradle.kts#L12) is an example of how to do this.

Reply:
i rewrite that.